versions.yaml

```diff
@@ -255,7 +255,7 @@ languages:
       description: |
         'newest-version' is the latest version known to work when
         building Kata
-      newest-version: "1.47.0"
+      newest-version: "1.54.0"
```

Comment:
I'm fine about this, but it may bite dowstreams depending on the version of rust they use on their buildroots & the version of kata-containers to be used on their products.

@dgibson, mind to doble-check what we use / will use on the Red Hat side? @sameo, @bergwolf, and other downstream consumers, it may be important for you as well.

Reply:
Also, https://github.com/kata-containers/kata-containers/pull/2409#issuecomment-894205494

Reply:
Note that this is only recording the latest tested Rust version, not the minimum required.  Also, I haven't finished looking into if there's additional changes required.

Reply:
The relevance of ##2409 is not obvious to me, sorry.

Reply:
There I mentioned:
```
I'd advise to also bump the minimal rust version used, but I'd recommend checking what's the version used downstream to ensure there's no issue. /cc @marcel-apf, @dgibson to ensure no issue is hit by bumping the version, if we take this path.
```

Reply:
So, I'm not really clear on what you're asking me to do or change here.  I don't see how this affects downstream: this doesn't limit what Rust versions you can use to build Kata, it only extends which versions (changing what we test and lint with in the meantime).

Reply:
Okay :-)